components/sdlutil/sdlgraphicswindow.hpp

```diff
@@ -80,6 +80,9 @@ class GraphicsWindowSDL2 : public osgViewer::GraphicsWindow
 
         SDL_Window *mWindow;
     };
+
+private:
+    void setSwapInterval(bool enable);
```

Comment:
Why don't place this function into anonymous namespace in .cpp file? It don't use any of this class fields and is not used outside `sdlgraphicswindow.cpp` file.

Reply:
The function "belongs" to the SDL window. Global state and stuff.